src/style.css

```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.first-block {
+  height: 100px;
```

Comment:
move this to separate class, and add it to all of the div's. Don't repeat yourself